### ISSUE TITLE
pinejs-client deprecated warning!

### ISSUE THREAD

**Issue by shaunmulligan** [resin-io/resin-cli#379]
when running the `resin devices` command, the CLI prints out a deprecation warning. We should update the pinejs-client as this error is very ugly and confusing.
```
 $ resin devices --app QuickTest
    	pinejs-client deprecated: `$filter: a: b: ...` is deprecated, please use `$filter: a: $any: { $alias: "x", $expr: x: b: ... }` instead.
    	ID 	UUID	NAME        	DEVICE TYPE  APPLICATION NAME STATUS IS ONLINE
    	192759 033ad44 falling-silence raspberrypi3 QuickTest    	Idle   true
```

**Comment by vitaly** [comment]
same here

**Comment by jviotti** [comment]
@shaunmulligan The problem is not that we need to update `pinejs-client`, but that some kind of query originating from the SDK is marked as deprecated by `pinejs-client`. I honestly don't understand the pine suggestion printed there, so I suggest moving this issue to the Resin SDK and bringing Page into the loop.

**Comment by lekkas** [comment]
Created issue and sent PR here: https://github.com/resin-io/resin-sdk/issues/223

**Comment by jviotti** [comment]
The fix landed in Resin SDK v5.3.6. I'm closing this issue since semver will make sure users get such version automatically.
